microsetta_public_api/api/diversity/alpha.py

```diff
@@ -14,8 +14,28 @@ def get_alpha(sample_id, alpha_metric):
     ret_val = {
         'sample_id': sample_id,
         'alpha_metric': alpha_data['alpha_metric'],
-        'data': alpha_data['data'][sample_id],
+        'data': alpha_data['alpha_diversity'][sample_id],
 
     }
 
     return jsonify(ret_val), 200
+
+
+def alpha_group(body, alpha_metric):
+    sample_ids = body['sample_ids']
+
+    alpha_repo = AlphaRepo()
+    if not all(alpha_repo.exists([sample_ids])):
+        return jsonify(error=404, text="Sample ID not found."), 404
```

Comment:
Could this send back which sample IDs were not found?

Reply:
Sure. See https://github.com/biocore/microsetta-public-api/pull/5/commits/69f40647910a9e07d7fe7e87690d95e7d58b2451